agents/superpowers/specs/2026-07-24-plugin-projects-design.md

```
@@ -0,0 +1,175 @@
+to extend it (via artifacts, templates) or use projects directly. It will be one
+of the core aspects of Composer.
+
+## Types
+
+### `Project` (`@dxos/compute`)
+
+Evolved in place from `Topic.ts` (Topic is deleted; no shims). Typename
+`org.dxos.type.project@0.2.0` — 0.2.0 disambiguates from stored instances of the
+name-squatting `@dxos/types` Project at 0.1.0.
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)